src/EnergyPlus/FluidProperties.hh

```diff
@@ -79,6 +79,12 @@ namespace FluidProperties {
     int constexpr iRefri = 1;
     int constexpr iGlyco = 1;
 
+    constexpr int DefaultNumGlyTemps(33);                  // Temperature dimension of default glycol data
+    constexpr int DefaultNumGlyConcs(10);                  // Concentration dimension of default glycol data
+    constexpr int DefaultNumSteamTemps(111);               // Temperature dimension of default steam data.
+    constexpr int DefaultNumSteamSuperheatedTemps(114);    // Temperature dimension of default steam data.
+    constexpr int DefaultNumSteamSuperheatedPressure(114); // Temperature dimension of default steam data.
```

Comment:
Moved these constants to FluidProperties.hh, so I can use them for initializing `std::array` sizes. Also changed them to `constexpr` to have them as compile-time constants.

Reply:
There are a lot of `NumCpTempPoints`, `NumCpConcPoints`, etc. variables in `FluidProperties`. It looks like those are all set to the default values and there is no path to set them to anything else. Does anyone know if those are true "variables" or whether they can simply be replaced with these `constexpr`'s? Several additional simplifications would be enabled if the latter was the case and this degree of freedom was unnecessary. Just asking.